src/components/Flexy/styles/Flexy.css.js

```diff
@@ -4,16 +4,26 @@ import styled from '../../styled'
 
 export const config = {
   gapSize: {
-    xs: 4,
-    sm: 8,
-    md: 12,
-    lg: 16,
-    xl: 20,
+    4: {
+      xs: 4,
+      sm: 8,
+      md: 12,
+      lg: 16,
+      xl: 20,
+    },
+    5: {
```

Comment:
Condition components use a different spacing scale. I didn't want to affect previous implementations, so I added a new base scale of `5` rather than adjusting the current numbers.